LineStream/XargsStream.test.ts

```diff
@@ -0,0 +1,54 @@
+import $ from "../mod.ts";
+import { assertEquals } from "../test_deps.ts";
+
+Deno.test("XargsStream async iterator", async () => {
+  // TODO: should xargs make stdout piped automaticly
```

Comment:
I think we should avoid automatically `stdout ("piped")`
because of the normal behavior of `echo "1\n2" | xargs -I {} echo {}{}` to write to stdout at run.